### ISSUE TITLE
Train on COCO Dataset

### ISSUE THREAD

**Issue by Strike1999** [UTokyo-FieldPhenomics-Lab/DODA#9]
Thanks for your great work!

I noticed that the training process in the README is based on the GWHD dataset. How can I train on the COCO dataset instead? Could you provide more details? Do I still need to follow the three-step training procedure?

Best Wishes

**Comment by illrayy** [comment]
Thank you for your interest in our work.

After downloading the dataset using [Download_dataset.sh](https://github.com/UTokyo-FieldPhenomics-Lab/DODA/blob/main/Download_dataset.sh),  then add `'train2017'` to [dataTypes](https://github.com/UTokyo-FieldPhenomics-Lab/DODA/blob/f624190fafc7cbff4c8644cdd53eeed43ebbf7f2/prepare_coco.py#L11) to get Layout images of training set.

Then change the `config` in [train.py](https://github.com/UTokyo-FieldPhenomics-Lab/DODA/blob/f624190fafc7cbff4c8644cdd53eeed43ebbf7f2/train.py#L11C15-L11C23) to [this](https://github.com/UTokyo-FieldPhenomics-Lab/DODA/blob/main/configs/DODA/coco_train.yaml) and you can start training.

For coco dataset, we used the stable-diffusion v1.5 as the pretrained weight, so the first two steps of training are not required. And you can download the weight [here](https://drive.google.com/file/d/1Z7Ix6wuDH1Em3gqL5ZKNQ3Hik_L9Rz2u/view?usp=sharing).

**Comment by Strike1999** [comment]
Your answers are very helpful. Thanks!

**Comment by Strike1999** [comment]
If I want to train a 512×512 resolution model on the COCO dataset, do I just need to change [coco_train.yaml](https://github.com/UTokyo-FieldPhenomics-Lab/DODA/blob/main/configs/DODA/coco_train.yaml)
`    train:
        size: 256`
`    validation:
        size: 256`
into
`    train:
        size: 512`
`    validation:
        size: 512`
？

**Comment by Strike1999** [comment]
Many thanks! By the way, is the learning rate kept the same when training the 512×512 model? And I need to change the `resume_path` to the 256×256 model I have trained.

**Comment by illrayy** [comment]
Yes, exactly

**Comment by Strike1999** [comment]
^_^
